cypress/integration/inventory/export.spec.js

```diff
@@ -0,0 +1,37 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryActions from '../../support/fragments/inventory/inventoryActions';
+import FileManager from '../../support/utils/fileManager';
+
+
+describe('inventory / actions: export UUIDs', () => {
+  beforeEach('navigates to Inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C9284 verifies export UUIDs instances', () => {
+    cy.do(InventorySearch.searchByEffectiveLocation());
+    InventoryActions.saveUUIDs();
+
+    // TODO: think about move it to separate func
+    cy.intercept('/search/instances/ids**').as('getIds');
+    cy.wait('@getIds').then((obj) => {
+      const expectedUUIDs = InventorySearch.getUUIDsFromRequest(obj);
+
+      // Need time for download file TODO: think about how it can be fixed
+      cy.wait(Cypress.env('downloadTimeout'));
```

Comment:
As indicated, It's a code-smell to have waits inside our tests. Instead, we should ask ourselves: "how will the user interface reflect the fact that this operation". For example:

```js
// SearchResults
cy.expect(SearchResults().exist())
```

Reply:
> "how will the user interface reflect the fact that this operation".

UI didn't affect by download process. We just click to action and then download starts

> cy.expect(SearchResults().exist())

I don't see how it can help. When download process starts chrome creates an empty file with id instead of full name and empty inside. This cy.expect can pick this file but I need full file after download ends. And I just don't know ho I can wait while file will downloaded fully. May be if you know, can you help with it please? It may be links or some examples
If you do, I will replace this code